conanfile.py

```diff
@@ -89,9 +105,12 @@ def package_info(self):
             self.cpp_info.libs = ['libx264.dll.lib' if self.options.shared else 'libx264']
             if self.options.shared:
                 self.cpp_info.defines.append("X264_API_IMPORTS")
-        elif self._is_mingw_windows:
+        elif self._is_mingw:
             self.cpp_info.libs = ['x264.dll' if self.options.shared else 'x264']
         else:
             self.cpp_info.libs = ['x264']
         if self.settings.os == "Linux":
             self.cpp_info.libs.extend(['dl', 'pthread', 'm'])
+        if self.settings.os == "Android":
+            self.cpp_info.libs.extend(['dl', 'm'])
+        self.cpp_info.names['pkg_config'] = 'x264.pc'
```

Comment:
```suggestion
        self.cpp_info.names['pkg_config'] = 'x264'
```

Without `.pc` I assume

Reply:
oops, wrong copy/paste.Thanks !